sunkit_pyvista/draw.py

```diff
@@ -0,0 +1,80 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import Longitude, SkyCoord
+from astropy.visualization.wcsaxes import Quadrangle
+
+from sunpy.coordinates.utils import get_rectangle_coordinates
+from sunpy.coordinates import HeliocentricInertial
+from sunpy.map.maputils import all_corner_coords_from_map
+
+from matplotlib.patches import Polygon
+from astropy.constants import R_sun
+
+
+__all__ = ['Draw']
+
+
+class Draw:
```

Comment:
I don't think I understand why this is a class? I think it would be best to get remove this for now, and just focus on getting the `plot_map` method working. We can add other stuff later once that bit is working.

Reply:
Ah, okay. 
I thought of splitting the code for generating pyvista meshes and the addition of them to the plotter separately